src/main/java/no/nav/fo/veilarbregistrering/enhet/adapter/Readme.md

```diff
@@ -0,0 +1,6 @@
+Enhetsregisteret
```

Comment:
Vet ikke om dette var lurt eller ikke, men tenkte kanskje det kunne være et naturlig sted for å dokumentere litt av løsningen vår, som så kan settes sammen i en helhet.

Kan f.eks. linke til Swagger-doc til Enhetsregisteret om vi finner den, og andre ting som utviklere etter oss kan tenkes å lure på.